driver-core/Upgrade_guide_to_2.0.rst

```diff
@@ -155,6 +155,13 @@ Other API Changes
     will now need to use Host#isUp and Cluster#register if you were using that
     class.
 
+11. LatencyTracker#update method now has a different signature and takes two new
+    parameters: the statement that has been executed (or null, if no
```

Comment:
The javadoc states that the statement parameter is never null, so this should change too.